README.md

```diff
@@ -36,6 +36,12 @@ To run properly, the iExec Blockchain Adapter API requires:
 | IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PATH | Path to the wallet of the server. | String | `src/main/resources/wallet.json` |
 | IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PASSWORD | Password to unlock the wallet of the server. | String | `whatever` |
 
+## Health checks
+
+A `/actuator/health` endpoint is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```

Comment:
```suggestion
An `/actuator/health` endpoint is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```

Reply:
I would use `A`, as JB does. We would pronounce it `a slash actuator [...]`, not `an slash actuator [...]`.

Reply:
Up to you guys :) (Slash is formatted at read time)

Reply:
What about this ?
```suggestion
An health endpoint (`/actuator/health`) is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```

Reply:
Done in 3787b79